cassandra/4.0/modules/ROOT/pages/cassandra-connector-studio.adoc

```diff
@@ -0,0 +1,127 @@
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP listener in the app.
+. On the *General* tab, specify the connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Apache Cassandra Connector, follow these steps:
+
+. In the *Mule Palette* view, select *CassandraDB* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. If an Apache Cassandra database has authentication enabled, you must provide credentials when accessing it from the connector. Apache Cassandra supports basic authentication only. To provide credentials to an Apache Cassandra database, enter the corresponding values in the Apache Cassandra Connector global element.
+
+To configure the global element for Apache Cassandra Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, complete the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field a|Description
+|Host | Host name, or IP address of an Apache Cassandra node
+|Port | Port number. The default port is `9042`
+|===
++
+If the target database has authentication enabled, you must also complete these fields:
++
+[%header%autowidth.spread]
+|===
+Field a|Description
+|Keyspace | Apache Cassandra keyspace
+|Username | Apache Cassandra user name
+|Password | Apache Cassandrad password.
+|===
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
++
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
```

Comment:
Can you please add here a description of the security tab as well?  I think I've put it only on design center doc. Thanks